ci/azure-test-stable.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         displayName: cargo ${{ parameters.cmd }} --all-features
         env:
           CI: "True"
+          RUST_TEST_THREADS: "1"
```

Comment:
Setting "linger" to 0 and forcibly terminating the connection can cause problems in other tests. Avoiding concurrency can help some.

Reply:
Let's just improve the test, I don't think this is a solution.